packages/client/lib/util/parse.ts

```diff
@@ -80,3 +88,8 @@ export function parseKey(input: string | Buffer) {
   }
   return Buffer.from(input, 'hex')
 }
+
+// From: https://community.helpsystems.com/forums/intermapper/miscellaneous-topics/5acc4fcf-fa83-e511-80cf-0050568460e4
```

Comment:
That is some intense regex.  I'm glad I didn't have to write it.

Reply:
ha ha, thats what i felt while i copied and tested it

Reply:
Short nit: this URL is forwarding to https://community.fortra.com/forums/intermapper/miscellaneous-topics/5acc4fcf-fa83-e511-80cf-0050568460e4 now (maybe they rebranded in the last few days?).

Maybe let's directly update the URL in case the old domain name goes over to some other (malicious) usage at some point (likely not, but who knows? 🙂).

Reply:
Anyhow, really a super-nitty nit. 😎